hera_sim/tests/test_vis.py

```diff
@@ -239,6 +239,10 @@ def test_autocorr_flat_beam(uvdata, simulator):
     )
     v = sim.simulate()
 
+    # Account for factor of 2 between Stokes I and 'xx' pol for vis_cpu
+    if simulator == VisCPU:
+        v *= 2.0
+
```

Comment:
This is the fix I guess -- I switched vis_cpu to applying a factor of 1/2 to the input source fluxes (assumed to be specified as Stokes I), as explained here: https://github.com/HERA-Team/vis_cpu/blob/main/src/vis_cpu/vis_cpu.py#L92
The rest of vis_cpu naturally deals with linear polarisation channels (rather than Stokes), so the output in this case is for the 'xx' polarisation and so needs to be multiplied by a factor of 2 again to get Stokes I.

This change makes the output consistent with pyuvsim, which you can see from this unit test in the vis_cpu package: https://github.com/HERA-Team/vis_cpu/blob/main/tests/test_compare_pyuvsim.py#L143

But in _this_ unit test (in hera_sim/tests/test_vis.py), the extra factor of 2 is needed, as I assume the other simulators are outputting Stokes I.

So, I'm not sure what the best course of action is here. I think the new vis_cpu behaviour makes sense, as it is now a polarised visibility simulator, and it matches pyuvsim.

Reply:
Hmm, OK if vis_cpu is consistent with pyuvsim, it probably means we instead need to update the HealVis wrapper to return results consistent with pyuvsim. This has been a persistent problem, and has not been helped by my own very vague understanding of polarization etc.

Reply:
Thanks, I can look into this if needed, but perhaps it's more sensible for the HealVis maintainer to do it (not sure who at the moment?)